src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -292,6 +292,12 @@ public HttpClientOptions setSsl(boolean ssl) {
   }
 
   @Override
+  public HttpClientOptions setSNIServerName(String serverName) {
```

Comment:
rename `setSniServerName` as the rest of the options tends to use non capital acronyms